controls/SV-204445.rb

```diff
@@ -1,57 +1,56 @@
-    If the file integrity application does not exist, or a script file controlling the execution of the file integrity
-    application does not exist, this is a finding.'
-  desc 'fix', 'Configure the file integrity tool to run automatically on the system at least weekly. The following
-    example output is generic. It will set cron to run AIDE daily, but other file integrity tools may be used:
-    # more /etc/cron.daily/aide
-    #!/bin/bash
-    /usr/sbin/aide --check | /bin/mail -s "$HOSTNAME - Daily aide integrity check run" root@sysname.mil'
+
+Note: A file integrity tool other than Advanced Intrusion Detection Environment (AIDE) may be used, but the tool must be executed at least once per week.
+
+Check for the presence of a cron job running daily or weekly on the system that executes AIDE daily to scan for changes to the system baseline. The command used in the example will use a daily occurrence.
+
+Check the cron directories for a script file controlling the execution of the file integrity application. For example, if AIDE is installed on the system, use the following command:
+
+     # ls -al /etc/cron.* | grep aide
+     -rwxr-xr-x 1 root root 602 Mar 6 20:02 aide
+
+     # grep aide /etc/crontab /var/spool/cron/root
+     /etc/crontab: 30 04 * * * root /usr/sbin/aide  --check
+     /var/spool/cron/root: 30 04 * * * /usr/sbin/aide  --check
+
+If the file integrity application does not exist, or a script file controlling the execution of the file integrity application does not exist, this is a finding.'
+  desc 'fix', 'Configure the file integrity tool to run automatically on the system at least weekly. The following example output is generic. It will set cron to run AIDE daily, but other file integrity tools may be used:
+
+     # more /etc/cron.daily/aide
+     #!/bin/bash
+
+     /usr/sbin/aide --check | /var/spool/mail -s "$HOSTNAME - Daily aide integrity check run" root@sysname.mil'
   impact 0.5
-  tag 'legacy': ['SV-86597', 'V-71973']
-  tag 'severity': 'medium'
-  tag 'gtitle': 'SRG-OS-000363-GPOS-00150'
-  tag 'gid': 'V-204445'
-  tag 'rid': 'SV-204445r603261_rule'
-  tag 'stig_id': 'RHEL-07-020030'
-  tag 'fix_id': 'F-36304r602622_fix'
-  tag 'cci': ['CCI-001744']
+  tag legacy: ['SV-86597', 'V-71973']
+  tag severity: 'medium'
+  tag gtitle: 'SRG-OS-000363-GPOS-00150'
+  tag gid: 'V-204445'
+  tag rid: 'SV-204445r880848_rule'
+  tag stig_id: 'RHEL-07-020030'
+  tag fix_id: 'F-36304r880847_fix'
+  tag cci: ['CCI-001744']
   tag nist: ['CM-3 (5)']
-  tag subsystems: ["file_integrity_tool"]
-  tag 'host', 'container'
+  tag subsystems: ['file_integrity_tool']
+  tag 'host'
+  tag 'container'
 
   file_integrity_tool = input('file_integrity_tool')
   file_integrity_interval = input('file_integrity_interval')
 
-  describe package(file_integrity_tool) do
-    it { should be_installed }
-  end
-
-  if file_integrity_interval == 'monthly'
+  if file_integrity_tool != 'aide'
+    describe 'The system is not set to ' do
```

Comment:
not sure this is a typo with the incomplete describe statement (and no test to finish the sentence off)

Reply:
I updated to be consistent with another.